deployer/scripts/hawkular.sh

```diff
@@ -187,12 +187,25 @@ EOF
   oc create -f templates/hawkular-cassandra-node-pv.yaml
   oc create -f templates/hawkular-cassandra-node-emptydir.yaml
   oc create -f templates/support.yaml
-  
+
   echo "Deploying Hawkular Metrics & Cassandra Components"
   oc process hawkular-metrics -v "IMAGE_PREFIX=$image_prefix,IMAGE_VERSION=$image_version,METRIC_DURATION=$metric_duration,MASTER_URL=$master_url" | oc create -f -
   oc process hawkular-cassandra-services | oc create -f -
-  oc process hawkular-support -v "HAWKULAR_METRICS_HOSTNAME=$hawkular_metrics_hostname" | oc create -f - || true
-  
+  oc process hawkular-support | oc create -f -
+
+  echo "Creating the Hawkular Metrics Route"
+  # We need to create the route after the service has been created so that its labels get applied to the route itself
+  route_params="--hostname=$hawkular_metrics_hostname --service=hawkular-metrics --dest-ca-cert=$dir/hawkular-metrics-ca.cert"
+  if [ -s ${secret_dir}/hawkular-metrics.pem ]; then
+    `openssl rsa -in  ${secret_dir}/hawkular-metrics.pem > $dir/custom-certificate.key`
+    `openssl crl2pkcs7 -nocrl -certfile ${secret_dir}/hawkular-metrics.pem | openssl pkcs7 -print_certs | grep -v "^subject=*\|^issuer=*\|^$" > $dir/custom-certificate.crt`
```

Comment:
looks like this is just extracting the cert from the pem, or am i missing something? seems like you could do that more simply:

    openssl x509 -in ${secret_dir}/hawkular-metrics.pem -out $dir/custom-certificate.crt

either way, could probably use a comment as to what this is for